functions/src/test/mocks.ts

```diff
@@ -238,7 +250,8 @@ function collectionGenerator(data: {[id: string]: Object}) {
             },
             collection: (i: string) => ({
                 add: (d: any) => Promise.resolve(d)
-            })
+            }),
+            delete: () => { exports.spyOnMe1(); Promise.resolve(true) }
```

Comment:
Same question, should the promise be returned?

Reply:
yes, collection.add returns a promise (you would normally use `await collection('x').add({ data })`)

Reply:
Oh I guess  you might be talking about delete - same thing there. In the real API, they're all async operations, so we are mimicking that API here.